index.d.ts

```diff
@@ -0,0 +1,177 @@
+export const GSS_C_DELEG_FLAG: number;
+export const GSS_C_MUTUAL_FLAG: number;
+export const GSS_C_REPLAY_FLAG: number;
+export const GSS_C_SEQUENCE_FLAG: number;
+export const GSS_C_CONF_FLAG: number;
+export const GSS_C_INTEG_FLAG: number;
+export const GSS_C_ANON_FLAG: number;
+export const GSS_C_PROT_READY_FLAG: number;
+export const GSS_C_TRANS_FLAG: number;
+
+export const GSS_C_NO_OID: number;
+export const GSS_MECH_OID_KRB5: number;
+export const GSS_MECH_OID_SPNEGO: number;
```

Comment:
Just leaving this comment in a random place, not related to this code but now's also a good time to _delete_ anything from the types that aren't public API (maybe you've already done that) just bringing it up to make sure. I imagine we'll bring over API extractor here so `@public` / `@internal` will do the shaving off of APIs we don't expect to be used externally.

Reply:
Sure! As long as the types have been correctly "broken" for the major release such that api-extractor adoption doesn't cause issues. Docs are nice to have, and might make it easier to not miss something that is meant to be public in the future migration

Reply:
Well, I meant more so that the d.ts file and the README are aligned in terms of our public exports.  

But regardless, done